package.json

```diff
@@ -60,8 +62,8 @@
     "chai-as-promised": "^5.0.0",
     "coveralls": "~2.11.2",
     "es6-promise": "^2.1.1",
-    "glob": "~5.0.5",
     "istanbul": "^0.3.13",
+    "jasmine": "^2.3.2",
```

Comment:
Jasmine should not be a dependecy (Mocha is because we use it to test the framework)